conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/020/package/.index.json

```diff
@@ -6,21 +6,21 @@
 			"resourceType": "ImplementationGuide",
 			"id": "hl7.fhir.us.davinci-hrex",
 			"url": "http://hl7.org/fhir/us/davinci-hrex/ImplementationGuide/hl7.fhir.us.davinci-hrex",
-			"version": "0.2.0"
+			"version": "1.0.0"
```

Comment:
i think we should change the package path to hl7/fhir/us/davinci-hrex/100/ to match

Reply:
done